Feb-07-2022 08:51:25 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-07-2022 08:52:34 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-07-2022 08:53:09 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-07-2022 08:54:13 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-07-2022 09:02:20 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-07-2022 09:10:13 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-08-2022 09:05:21 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-13-2022 15:25:22 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:15)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:16)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:52:35
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:50:19
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Feb-21-2022 23:13:01 | [33mwarn[39m : request to http://localhost:8080/api/users/save failed, reason: connect ECONNREFUSED 127.0.0.1:8080
Feb-21-2022 23:14:43 | [31merror[39m : Error: 400. Message: "username" is required. Stack: Error: "username" is required
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-21-2022 23:15:07 | [31merror[39m : Error: 400. Message: "username" is required. Stack: Error: "username" is required
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-21-2022 23:15:59 | [31merror[39m : Error: 400. Message: "username" is required. Stack: Error: "username" is required
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-22-2022 08:58:57 | [33mwarn[39m : mockedFetch.mockImplementationOnce is not a function
Feb-22-2022 08:59:47 | [33mwarn[39m : mockedFetch.mockImplementationOnce is not a function
Feb-22-2022 09:00:09 | [33mwarn[39m : mockedFetch.mockImplementationOnce is not a function
Feb-22-2022 22:07:32 | [33mwarn[39m : mockedFetch.mockImplementationOnce is not a function
Feb-22-2022 22:56:20 | [33mwarn[39m : Cannot read property 'json' of undefined
Feb-22-2022 23:00:58 | [33mwarn[39m : Cannot read property 'json' of undefined
Feb-23-2022 22:14:04 | [33mwarn[39m : mockedAxios.mockImplementationOnce is not a function
Feb-23-2022 22:14:05 | [31merror[39m : Error: 400. Message: You must provide a valid date.. Stack: Error: You must provide a valid date.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-23-2022 22:15:18 | [33mwarn[39m : mockedAxios.mockImplementationOnce is not a function
Feb-23-2022 22:15:18 | [31merror[39m : Error: 400. Message: You must provide a valid date.. Stack: Error: You must provide a valid date.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-23-2022 22:15:18 | [33mwarn[39m : Request failed with status code 400
Feb-23-2022 22:17:37 | [31merror[39m : Error: 400. Message: You must provide a valid date.. Stack: Error: You must provide a valid date.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-23-2022 22:19:43 | [31merror[39m : Error: 400. Message: You must provide a valid date.. Stack: Error: You must provide a valid date.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Feb-24-2022 16:51:52 | [31merror[39m : Error: 400. Message: You must provide a valid date.. Stack: Error: You must provide a valid date.
    at new ApiError (C:\Users\DEPOSITO\Desktop\LAUTARO\Coder\ecommerce-project\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (C:\Users\DEPOSITO\Desktop\LAUTARO\Coder\ecommerce-project\server\src\api\errorApi.ts:23:12)
    at C:\Users\DEPOSITO\Desktop\LAUTARO\Coder\ecommerce-project\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\DEPOSITO\Desktop\LAUTARO\Coder\ecommerce-project\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
Mar-02-2022 13:51:18 | [33mwarn[39m : [2mexpect([22m[31mreceived[39m[2m).[22mtoEqual[2m([22m[32mexpected[39m[2m) // deep equality[22m

[32m- Expected  - 15[39m
[31m+ Received  +  0[39m

[33m@@ -1,8 +1,7 @@[39m
[2m  Object {[22m
[2m    "data": Object {[22m
[32m-     "data": Object {[39m
[2m      "_id": "621fa086c3c19196b3d03ad2",[22m
[2m      "age": "12-27-2000",[22m
[2m      "avatar": "",[22m
[2m      "createdAt": "2022-03-02T13:51:18-03:00",[22m
[2m      "facebookID": "",[22m
[33m@@ -14,20 +13,6 @@[39m
[2m      "repeatedPassword": "test123",[22m
[2m      "surname": "BOT",[22m
[2m      "username": "test@gmail.com",[22m
[2m    },[22m
[2m    "message": "User successfully created.",[22m
[32m-   },[39m
[32m-   "headers": Object {[39m
[32m-     "access-control-allow-credentials": "true",[39m
[32m-     "access-control-allow-origin": "http://localhost:3000",[39m
[32m-     "connection": "close",[39m
[32m-     "content-length": "response lenght",[39m
[32m-     "content-type": "application/json; charset=utf-8",[39m
[32m-     "date": "response date",[39m
[32m-     "etag": "request etag",[39m
[32m-     "vary": "Origin",[39m
[32m-     "x-powered-by": "Express",[39m
[32m-   },[39m
[32m-   "status": 201,[39m
[32m-   "statusText": "Created",[39m
[2m  }[22m
Mar-02-2022 13:59:31 | [31merror[39m : Error: 400. Message: "username" is required. Stack: Error: "username" is required
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-02-2022 13:59:31 | [31merror[39m : Request failed with status code 400
Mar-02-2022 14:00:01 | [31merror[39m : Error: 400. Message: "username" is required. Stack: Error: "username" is required
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\users.ts:44:30
    at Generator.next (<anonymous>)
    at fulfilled (D:\ESTUDIO\ecommerce\server\src\controller\users.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:22:25 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:34:34
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:30:13
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Mar-03-2022 12:23:15 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:143:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:27:10 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:143:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:28:37 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:143:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:32:07 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:143:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:39:21 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:144:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:41:36 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:144:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:42:37 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:144:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:44:06 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:144:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 12:56:01 | [31merror[39m : Error: 500. Message: An error occured.. Stack: Error: An error occured.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.internalError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:29:12)
    at MongoOrders.<anonymous> (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:144:23)
    at Generator.throw (<anonymous>)
    at rejected (D:\ESTUDIO\ecommerce\server\src\models\DAOs\Mongo\orders.ts:6:65)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mar-03-2022 14:40:34 | [31merror[39m : An error occured
Mar-03-2022 14:40:34 | [31merror[39m : An error occured
Mar-03-2022 14:45:45 | [31merror[39m : An error occured
Mar-03-2022 14:45:45 | [31merror[39m : An error occured
Mar-03-2022 23:40:31 | [31merror[39m : An error occured
Mar-03-2022 23:40:32 | [31merror[39m : An error occured
Mar-03-2022 23:46:09 | [31merror[39m : An error occured
Mar-03-2022 23:46:09 | [31merror[39m : An error occured
Mar-03-2022 23:49:15 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:34:34
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:30:13
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Mar-03-2022 23:50:04 | [31merror[39m : An error occured
Mar-03-2022 23:50:04 | [31merror[39m : An error occured
Mar-03-2022 23:56:58 | [31merror[39m : An error occured
Mar-03-2022 23:58:04 | [31merror[39m : An error occured
Mar-04-2022 00:05:45 | [31merror[39m : invalid_grant
Mar-04-2022 00:06:33 | [31merror[39m : invalid_grant
Mar-04-2022 00:07:20 | [31merror[39m : invalid_grant
Mar-04-2022 00:07:46 | [31merror[39m : invalid_grant
Mar-04-2022 00:12:05 | [31merror[39m : putoo
Mar-04-2022 00:12:05 | [31merror[39m : An error occured.
Mar-04-2022 00:56:17 | [31merror[39m : invalid_grant
Mar-04-2022 00:56:30 | [31merror[39m : invalid_grant
Mar-04-2022 00:56:32 | [31merror[39m : invalid_grant
Mar-04-2022 00:57:46 | [31merror[39m : invalid_grant
Mar-04-2022 01:01:07 | [31merror[39m : invalid_grant
Mar-04-2022 01:06:56 | [31merror[39m : invalid_grant
Mar-04-2022 01:07:23 | [31merror[39m : invalid_grant
Mar-04-2022 01:07:30 | [31merror[39m : invalid_grant
Mar-04-2022 01:07:30 | [31merror[39m : invalid_grant
Mar-04-2022 01:12:46 | [31merror[39m : invalid_grant
Mar-04-2022 01:12:52 | [31merror[39m : invalid_grant
Mar-04-2022 01:22:11 | [31merror[39m : Error: 400. Message: You must be logged in.. Stack: Error: You must be logged in.
    at new ApiError (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:14:11)
    at Function.ApiError.badRequest (D:\ESTUDIO\ecommerce\server\src\api\errorApi.ts:23:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:34:34
    at module.exports (D:\ESTUDIO\ecommerce\server\node_modules\jsonwebtoken\verify.js:53:12)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:30:13
    at Generator.next (<anonymous>)
    at D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:4:12)
    at login (D:\ESTUDIO\ecommerce\server\src\controller\auth.ts:21:16)
Mar-04-2022 01:23:02 | [31merror[39m : invalid_grant
